pyccel/parser/syntactic.py

```diff
@@ -621,8 +623,16 @@ def _visit_FunctionDef(self, stmt):
         imports      = []
 
         # TODO improve later
```

Comment:
I don't know what this TODO actually means.

@rmahjoubi If you think that your changes are a sufficient improvement here, I suggest you remove the TODO comment.

Reply:
There's still the issue about storing duplicated decorators

Reply:
A TODO comment should contain enough information for another developer to understand what needs doing. Either this comment needs more info. Or it should just refer to the issue. Or it should be removed and the issue is itself sufficient

Reply:
Ok, I will remove it